### PR TITLE
Fluentd updates

### PR DESCRIPTION
**Pull request by jchengsfx**

### CODE REVIEW

File: internal/buildscripts/packaging/fpm/etc/otel/collector/fluentd/conf.d/tomcat.conf [hunk 1]
```diff
@@ -17,7 +17,6 @@
     format_firstline /^(\w+\s\d+,\s\d+)|(\d+-\d+-\d+\s)/
     format1 /(?<message>.*)/
   </parse>
-  format multiline
```
Comment (flands) [inline internal/buildscripts/packaging/fpm/etc/otel/collector/fluentd/conf.d/tomcat.conf:20]: Is this not needed?

**Review by flands — APPROVED** [review]
Looks good -- one question

File: internal/buildscripts/packaging/fpm/etc/otel/collector/fluentd/conf.d/tomcat.conf (hunk 1 above)
Comment (jchengsfx) [reply internal/buildscripts/packaging/fpm/etc/otel/collector/fluentd/conf.d/tomcat.conf:20]: According to https://docs.fluentd.org/input/tail#format, it is deprecated.  I assume that the parser section you already have in there should handle multilines.  Also the fluentd log says that it is ignored if left in.

Comment (flands) [reply internal/buildscripts/packaging/fpm/etc/otel/collector/fluentd/conf.d/tomcat.conf:20]: oh this is format which is deprecated -- good call.

**Pull request merged**